content/manuals/copilot/examples.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Example prompts for the Docker agent
+linkTitle: Example prompts
+description: |
+  Discover example prompts to interact with the Docker agent and learn how to
+  automate tasks like Dockerizing projects or opening pull requests.
+weight: 30
+---
+
+{{% restricted title="Early Access" %}}
+The Docker for GitHub Copilot extension is an [early access](/release-lifecycle#early-access-ea) product.
+{{% /restricted %}}
+
+Here are some examples of the types of questions you can ask the Docker agent:
+
+### Ask general Docker questions
+
+You can ask general question about Docker. For example:
+
+- `@docker what is a Dockerfile`
+- `@docker how do I build a Docker image`
+- `@docker how do I run a Docker container`
+
+### Ask questions about your project
+
+You can ask questions about your project, such as:
+
+- `@docker what is the best way to Dockerize this project`
+- `@docker can you help me find vulnerabilities in my project?`
+
+The Docker agent will analyze your project, generate the necessary files, and,
+if applicable, offer to raise a pull request with the necessary Docker assets.
+
+## Performing actions on your behalf
```

Comment:
Maybe it's be worth explaining that only the containerization feature can open a PR right now. That could change down the line, but the current status is that the user can't explicitly ask the agent to open a PR with whatever files the agent might have generated. Definitely worth looking into that as a future feature though